R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -167,6 +167,18 @@ test_that("spark.lapply should perform simple transforms", {
   sparkR.session.stop()
 })
 
+test_that("add jar should work and allow usage of the jar on the driver node", {
+  sparkR.sparkContext()
+
+  destDir <- paste0(tempdir(), "/", "testjar")
+  jarName <- callJStatic("org.apache.spark.TestUtils", "createDummyJar",
```

Comment:
I checked what it's got right after `createDummyJar` call. It was `createDummyJar:C:\Users\appveyor\AppData\Local\Temp\1\Rtmpg5tOYo/testjar`